.changeset/real-colts-nail.md

```diff
@@ -0,0 +1,5 @@
+---
+'@blocto/blocto-wagmi-connector': major
```

Comment:
should be `minor`?

Reply:
same opinion. 
FYI redo just delete this file and add changeset again.

Reply:
is able to replace the string `major` to `minor` ?